demovieworderhooks/src/Entity/PackageLocation.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * 2007-2020 Friends of PrestaShop
```

Comment:
```suggestion
 * 2007-2020 PrestaShop
```
This demo module will not be hosted on http://github.com/friends-of-prestashop/